targz_test.go

```diff
@@ -222,12 +222,16 @@ func createTestData() (string, string) {
 		panic(err)
 	}
 
-	_, err = os.Create(filepath.Join(subDirectory, "my_file.txt"))
-	if err != nil {
+	if err := os.WriteFile(filepath.Join(subDirectory, "my_file.txt"), []byte("file contents"), 0666); err != nil {
```

Comment:
This change is here so that the tests do not accidentally pass as all file's sizes are `0` in the previous suite.